src/javascript/class-scanner.ts

```diff
@@ -586,6 +586,80 @@ class CustomElementsDefineCallFinder implements Visitor {
   }
 }
 
+export function extractPropertyFromMethod(
```

Comment:
These functions aren't used outside this file, so they should not be exported.

Every declaration should be as private and as read only as possible, IMO.